examples/rollup/README.md

```diff
@@ -21,8 +21,13 @@ pip3 install -r requirements.txt
 
 ```
 Run 
-yarn local-test-install --> download packages from yalc cache
-yarn start --> runs rollup to compile and starts the server
+yarn install # installs all packages
+yarn start # runs rollup to compile and starts the server
 ```
 
 3. Go to [http://localhost:4242/index.html](http://localhost:4242/index.html)
+
+Note: if you are doing local development, you can use yalc to test local changes
+
+1. yalc add @stripe/connect-js # in this sample
+2. yarn build && yalc push #from connect-js after you've made changes 
```

Comment:
I think this isn't accurate, these sound more correct to me. Can you `yalc add` without `yalc publish` first?

```suggestion
1. yarn build # from the root of this repository
1. yalc publish # from the root of this repository
2. yalc add @stripe/connect-js # in this sample
3. Repeat this process whenever there are any changes to the package
```